### PR TITLE
Update documents

### LINKED ISSUE #453: Update installation guide to latest
- Parents: #451 

---

## Checklist

- [ ] New Feature for the Service/Plugin (Vatz)
- [ ] Enhancement (Vatz)
- [x] others(etc. e.g, documentation, project policy management)

---

Since we have updated our document for installation in beginning of this year. 
It's already half year that hasn't updated yet. 

Update Installation guide as it and all other documents as latest.

### PR DESCRIPTION
**Pull request by qkrybin**
### 1. Type of change

Please delete options that are not relevant.

- [ ] New feature
- [ ] Enhancement
- [ ] Bug/fix (non-breaking change which fixes an issue)
- [x] others (anything other than above)

---

**Related:** #447 
close #453 

**Summary**
- update installation.md
- create community_plugins.md

---

### CODE REVIEW

File: docs/community_plugins.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+You could remove it via PR, but only if the person who added it to the list requests to be removed from the list.
+
+> ### 1. Plugin Name
+> 1. Summary: Describe what is your plugin for
+> 2. Github: Github address of your plugin repository 
+> 3. Version: https://pkg.go.dev link for latest version.
+> 4. Usage: Add link to a guide on how to intall and use your plugin.
+---
+## List
+### 1. vatz-plugin-sysutil (example)
```
Comment (xellos00) [inline docs/community_plugins.md:13]: @qkrybin 
Thanks for including this on documents
Can you update plugin name that is listed on 
- https://github.com/dsrvlabs/vatz/pull/457
   - vatz-plugin-sysutil
      - vatz_cpu_monitor
      - vatz_mem_monitor
      - vatz_disk_monitor
      - vatz_net_monitor
   - vatz-cosmos-hub
      - vatz_block_sync
      - vatz_node_is_alived
      - vatz_peer_count
      - vatz_active_status
      - vatz_gov_alarm

Comment (qkrybin) [reply docs/community_plugins.md:13]: updated

File: docs/installation.md [hunk 2]
```diff
@@ -3,48 +3,55 @@
...
 
 ## 2. VATZ Plugin
 
-1. Copy the plugin address
-  > **Plugin Repositories**
+### 1. Copy the plugin address
+
+  We officially provide 3 plugins for System Utility and 5 plugins for CosmosHub node. 
+  > **Official Plugin Repositories**
   > - [vatz-plugin-sysutill](https://github.com/dsrvlabs/vatz-plugin-sysutil)
```
Comment (gnongs) [inline docs/installation.md:39]: @qkrybin I think it need to fix typo.

Comment (qkrybin) [reply docs/installation.md]: fixed!

**Review by gnongs — APPROVED** [review]
LGTM!

File: docs/installation.md [hunk 3]
```diff
@@ -3,48 +3,60 @@
...
-  > - [vztz-plugin-cosmoshub](https://github.com/dsrvlabs/vatz-plugin-cosmoshub)
-  >   - node_active_status
-  >   - node_block_sync
-  >   - node_governance_alarm
-  >   - node-is_alived
-  >   - node_peer_count
+### 1. Copy the plugin address
 
-  We officially provide 3 plugins for System Utility and 5 plugins for CosmosHub node. But you can easily develop custom plugins for the feature that you want with [VATZ SDK](https://github.com/dsrvlabs/vatz/tree/main/sdk).
+  We officially provide 3 plugins for System Utility and 5 plugins for CosmosHub node.
```
Comment (heejin-github) [inline docs/installation.md:42]: We have 4 plugins for System Utility now.

File: docs/installation.md [hunk 4]
```diff
@@ -113,13 +142,18 @@ plugins_infos:
...
-1. `port`: check your machine used port number. If you set it to a port number that is already in use, an error will occur.
+1. `port`: Check your machine used port number. If you set it to a port number that is already in use, an error will occur.
 2. `health_checker_schedule`: Use cron expression to set the time to check if vatz is alive. 
-3. `dispatch_channels`: Add infos to receive alerts through discord, pagerduty, and telegram. 
-4. `reminder_schedule`: Use cron expression to set the time to resend the alert if it is not confirmed.
+3. `notificaiton_info`:
+   - `dispatch_channels`: Add infos to receive alerts through discord, pagerduty, and telegram. 
+   - `reminder_schedule`: Use cron expression to set the time to resend the alert if it is not confirmed.
+4. `rpc_info`: Check address and port to connect with plugins.
+5. `monitoring_info`: Check address and port to use monitoring. Currently, VATZ supports monitoring only through Prometheous-Grafana.
```
Comment (heejin-github) [inline docs/installation.md:151]: `Prometheous` -> `Prometheus`

**Review by heejin-github — CHANGES_REQUESTED** [review]

File: docs/installation.md (hunk 3 above)
Comment (qkrybin) [reply docs/installation.md]: fixed

File: docs/installation.md (hunk 4 above)
Comment (qkrybin) [reply docs/installation.md]: fixed

Comment (xellos00) [reply docs/installation.md]: @heejin-github 
Can you resolve this?

**Review by heejin-github — APPROVED** [review]
LGTM

**Pull request merged**